fastlane/Fastfile

```diff
@@ -141,7 +141,7 @@ platform :ios do
     generate_strings_file_for_glotpress
 
     UI.confirm('Ready to push changes to remote and trigger the beta build?') unless ENV['RELEASE_TOOLKIT_SKIP_PUSH_CONFIRM']
-    push_to_git_remote  
+    push_to_git_remote(tags: false)
```

Comment:
I wonder why fastlane developers opted to [make `tags` parameter `true` by default](https://docs.fastlane.tools/actions/push_to_git_remote/). That seems a bit risky to me. 🤷

Reply:
Yep I was surprised by this as well, quite unfortunate.

I'm back-and-forth on creating our _own_ helper action (which could be something like `git_commit(message:files:push:)` or something like that) in the release-toolkit to have our own default values and options, also because this built-in one as that `tags: true` as default. Not sure if it's worth it though.

Reply:
I don't know how often this method is used, so I don't have any feedback on that. However, what you're proposing makes sense to me if you feel it'd be worthwhile.

Not so fun fact, I struggled a bit finding where this method(?) was defined. I searched for the project and `release-toolkit` convinced that it was supposed to be there. I only later realized it might be a fastlane method. One of many things I struggle with Ruby(?) or the release-toolkit structure where things are not very obvious for someone like me who is used to code being a bit more explicit.

Reply:
Doesn't really answer your question, but when the parameter was added, they set it to `true` for backward compatibility.

I think it would be reasonable to suggest to switch to `false` as the default. I wonder how they would treat that change from a SemVer point of view, though 🤔 Maybe there's too much inertia in the tool's adoption to introduce a breaking but little change like this

Reply:
I'm pretty sure from how the fastlane team runs things that this would be a no-no for them to change the default value of this action there, because they're pretty conservative in the API, oftentimes preferring to keep an old API intact despite finding flaws in it later, rather than doing a breaking change.

Since it was created and reach its first stable version (1.0.0) years, fastlane only made a major version (2.x.x) once (!), especially because of that and the fact they really seem to try very hard to avoid doing any breaking changes (in order to not have users make migrations every N months or so). So changing the default value here is very likely out of the question for them.

> Not so fun fact, I struggled a bit finding where this method(?) was defined. I searched for the project and release-toolkit convinced that it was supposed to be there. I only later realized it might be a fastlane method. One of many things I struggle with Ruby(?) or the release-toolkit structure where things are not very obvious for someone like me who is used to code being a bit more explicit.

💯  I wish Fastlane had at least some namespacing concept, so that we could do `a8c.some_action` to distinguish our actions from the Release Toolkit from the rest. Or even better, an ability to have nested namespaces like `a8c.ios.some_action`, `a8c.ios.git.some_sub_action`… That would help clarify a lot. Sadly I don't think that's a feature they currently support 😒 

PS: Note that Fastlane now supports writing `Fastfiles` in Swift (as an alternative to Ruby). Which might (?) provide us with nicer type checking and API in our Fastfiles (though given the dynamic nature of the available actions with the plugin system, not sure how much it would improve things in practice…). I am guessing that this means depending on the presence of the Swift compiler (Xcode or the CommandLine Tools) being installed on the Mac running `fastlane`, which is ok for iOS projects but not an ideal for Android projects…